### PR TITLE
Enable prettier on frontend/

### PR DESCRIPTION
**Pull request by spadgett**
No time like the present.

/cc @christianvogt

### CODE REVIEW

**Comment by christianvogt** [comment]
Are you going to try and remove the eslint rules from the root now too? Or leave that for a followup pr?

**Comment by spadgett** [comment]
> Are you going to try and remove the eslint rules from the root now too? Or leave that for a followup pr?

For now, I've only removed the rules that duplicate prettier. I thought we'd work on additional changes in follow-up PRs.

**Comment by christianvogt** [comment]
/lgtm

A moment of silence for all those pending PRs.

**Comment by spadgett** [comment]
Rebased (I might need to hold other component/core PRs to get this through...)

File: frontend/.vscode/settings.json [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
-  //   "editor.formatOnSave": true
-  // },
-  // "[typescript]": {
-  //   "editor.formatOnSave": true
-  // },
-  // "[typescriptreact]": {
-  //   "editor.formatOnSave": true
-  // },
+  "[javascript]": {
+    "editor.formatOnSave": true
```
Comment (spadgett) [inline frontend/.vscode/settings.json]: @christianvogt fyi, i went ahead and uncommented these lines to always enable prettier

**Comment by spadgett** [comment]
I've put a hold on other component/core PRs in the queue until this merges.

**Pull request merged**